src/Compilers/VisualBasic/Portable/Symbols/SubstitutedNamedType.vb

```diff
@@ -663,6 +663,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 End Get
             End Property
 
+            Friend NotOverridable Overrides ReadOnly Property TypeArgumentsCustomModifiers As ImmutableArray(Of ImmutableArray(Of CustomModifier))
+                Get
+                    Return ArrayBuilder(Of ImmutableArray(Of CustomModifier)).GetInstance(Arity, ImmutableArray(Of CustomModifier).Empty).ToImmutableAndFree()
```

Comment:
I will move it into a helper function, but I will not add caching until perf numbers indicate it is needed.